src/main/resources/templates/crossref/body.ftlx

```diff
@@ -13,6 +13,9 @@
 				<publication_date media_type="online">
 					<year>${pub.publicationYear}</year>
 				</publication_date>
+				<publisher_item xmlns="http://www.crossref.org/schema/4.4.2">
```

Comment:
Why is there an extra namespace declaration here? It should be at the top of the document

Reply:
good catch, taken out.